test/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys, types
+sys.modules['pandas'] = types.ModuleType('pandas', 'Fake pandas module')
```

Comment:
Also, note that pandas is a requirement in setup.py UNLESS the environment indicates it's travis-ci or another CI. So installing and running the tests locally will still check for pandas requirement, and import the genuine pandas module.

If the test code ever grows tests that actually call pandas code, our choices seem to be:

1. Stop using travis-ci, which will make support for multiple Python versions hard to maintain.
2. Upgrade to a paid travis-ci account, or wait until Travis offers the dependency caching feature to open-source projects.
3. Mock the portions of the pandas module/contents that are covered by the tests, and use those mocks only in CI mode.

I would vote for #3.